apps/hobbies-helsinki/.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+## Description
```

Comment:
.github folder probably should be merged to the root folder.

Reply:
fixed in #23